js/viz/core/utils.js

```diff
@@ -444,6 +444,50 @@ export function getVizRangeObject(value) {
     }
 }
 
+export function normalizeArcParams(x, y, innerR, outerR, startAngle, endAngle) {
```

Comment:
What is `r`? Radius? Can we write a full name?

Reply:
What does it mean "renaming"? I still see strange abbreviation although you close the conversation